application/server/server.go

```diff
@@ -142,32 +141,25 @@ func codeLensHandler() jrpc2.Handler {
 	return handler.New(func(ctx context.Context, params sglsp.CodeLensParams) ([]sglsp.CodeLens, error) {
 		c := config.CurrentConfig()
 		c.Logger().Info().Str("method", "CodeLensHandler").Msg("RECEIVING")
-		defer c.Logger().Info().Str("method", "CodeLensHandler").Msg("SENDING")
 
 		lenses := codelens.GetFor(uri.PathFromUri(params.TextDocument.URI))
 
 		// Do not return Snyk Code Fix codelens when a doc is dirty
 		isDirtyFile := di.FileWatcher().IsDirty(params.TextDocument.URI)
+
+		defer c.Logger().Info().Str("method", "CodeLensHandler").
+			Bool("isDirtyFile", isDirtyFile).
+			Int("lensCount", len(lenses)).
+			Msg("SENDING")
+
 		if !isDirtyFile {
 			return lenses, nil
 		}
-
-		return filterCodeFixCodelens(lenses), nil
```

Comment:
I removed the return of "non-fix" code lenses - it doesn't make sense to return them either, limiting it to code is not sensible. Also, we don't show any other code lenses than fix code lenses right now, so it should not have visible effects for now.